.changelog/unreleased/bug/747-fix-747-Protobuf-out-of-memory.md

```diff
@@ -0,0 +1,2 @@
+- Tendermint ConsensusState -> Any can crash if out of memory
```

Comment:
Would you also add a unclog for the own PR under `breaking-changes`, so we can track the `encode_vec` signature change too?

Reply:
fixed